changelog/1.16.0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "1.16.0"
+description: "Released on 02/22/2021"
+---
+
+### Breaking Changes ❗
+
+- The deprecated `Services` feature is now **removed**. All resources created
+  for Services are preserved for data recovery purposes (volumes,
+  database tables, etc.), but are not accessible through the Coder platform.
+  Please reference
+  [Docker in Environments](../environments/cvms) for
+  a guide on the new workflow for running containers with your Environment.
+- The built-in Internal Extension Marketplace is now **removed**. All resources
+  created for this feature are preserved for data recover purposes, but are not
+  accessible through the Coder platform. For air-gapped deployments, the `Custom`
+  Extension Marketplace configuration enables the use of
+  [Open VSX](https://github.com/eclipse/openvsx) as
+  an alternative. For more details, please view our [migration guide](TODO).
+- The administrator configuration option "Privileged Environments" is now
+  removed. All non-CVM environments are launched with `privileged: false`.
+- The [Container-based Virtual Machine](../admin/environment-management/cvms)
+  environment option is now considered generally available. The
+  admin-level configuration is still required, but the `alpha`
+  warning has been removed.
+- Environment SSH routing has new behavior. Please reference
+[this guide](../admin/environment-management/ssh-access.md).
+
+### Features ✨
+
+- Public Rest API <apidocs.coder.com>
+- Support for remote port forwarding over SSH
+- Support for [running a full OpenSSH server](../admin/environment-management/ssh-access.md)
+inside your Environment.
+- web: PWA icons for jetbrains and code-server have been redesigned
+- web: Input sliders have been redesigned
+- web: A character counter was added to the organization description field
+- web: Display warning when memory provision rate is high
```

Comment:
Wasn't the dash redesigned in this update too? I'm happy to contribute the screenshots and such for those changes.

Reply:
We'll have screenshots in the blog post, if we want to include in the changelog too, feel free to use any in https://drive.google.com/drive/folders/1xHEmun0pAphFM3FDwlhR_JP_eIXHDM5v https://drive.google.com/drive/folders/1xHEmun0pAphFM3FDwlhR_JP_eIXHDM5v

Reply:
we can mention that it's been updated

Reply:
```suggestion
- web: Display warning when memory provision rate is high
- web: The dashboard has been redesigned with a new color scheme and navbar branding
```